README.md

```diff
@@ -10,6 +10,20 @@ It is targeted to run on the raspberry pi zero, b+ and 2.
 
 [Build parameters](kernel/README.md)
 
+## Docker
+One option to run the project is to make use of the [Docker](https://www.docker.com/) image. To do so you first need to download Docker and Docker Compose for your operating system, after which you can run the follow commands to build and start the container:
+
+```bash
+docker-compose build
+docker-compose up -d
+```
+
+After this you can choose to either open the shell to the container by running:
```

Comment:
either?

Reply:
maybe also document how you can now run qemu in this shell. I know, but new users (next year and this year) may not.

Reply:
It would probably be the best to link to the relevant README section to prevent info desync

Reply:
perfect.